### DISCUSSION TITLE
Why webpack5 cache configuration  is not work

### DISCUSSION THREAD

**Discussion by zyprepare** [webpack/webpack#16564] [Q&A]
This is my configuration item:

`cache: {
  type: 'filesystem', 
  buildDependencies: {
    config: [__filename]
  }
}`

But no cache file is generated in the. cache directory.

**Comment by alexander-akait** [comment]
Why are you thinking it is not working? What is console output? Where are you looking for file?

**Reply by zyprepare** [reply]
Thank you! Now the problem has been solved.
The reason is that I use the node api and need to call compiler.close. https://github.com/webpack/webpack/issues/12345
